images/virtualization-artifact/pkg/controller/vm/internal/statistic.go

```diff
@@ -52,15 +55,144 @@ func (h *StatisticHandler) Handle(ctx context.Context, s state.VirtualMachineSta
 	if err != nil {
 		return reconcile.Result{}, err
 	}
-
 	h.syncStats(current, changed, kvvmi)
+
+	pods, err := s.Pods(ctx)
+	if err != nil {
+		return reconcile.Result{}, err
+	}
+	pod, err := s.Pod(ctx)
```

Comment:
What is the difference between pod and pods?

Reply:
Pods - all pods of vm
Pod - active pod